docs/reference/how-to/fix-common-cluster-issues.asciidoc

```diff
@@ -1,7 +1,104 @@
 [[fix-common-cluster-issues]]
 == Fix common cluster issues
 
-This guide describes how to fix common problems with {es} clusters.
+This guide describes how to fix common errors and problems with {es} clusters.
+
+[discrete]
+=== Error: disk usage exceeded flood-stage watermark, index has read-only-allow-delete block
+
+This error indicates a data node is critically low on disk space and has reached
+the <<cluster-routing-flood-stage,flood-stage disk usage watermark>>. When a
+node reaches this watermark, {es} blocks writes to any index with a shard on the
+node to prevent a full disk. If the block affects related system indices, {kib}
+and other {stack} features may become unavailable.
+
+{es} will automatically remove the write block when the affected node goes below
+the <<cluster-routing-watermark-high,high disk watermark>>. To achieve this,
+{es} automatically moves some of the affected node's shards to other nodes in
+the data tier. However, {es} will only allocate shards to nodes below the high
+disk usage watermark.
+
+To verify that some nodes in the tier are below the high disk watermark, use the
+<<cat-allocation,cat allocation API>> to get each node's disk usage percentage
+(`disk.percent`) and current available disk space (`disk.avail`).
+
+[source,console]
+----
+GET _cat/allocation?v=true
+----
+
+To check that shards are moving from the affected node to other nodes, use the
+<<cat-shards,cat shards API>>. If {es} can't allocate a shard to another node,
+the shard has a `state` of `UNASSIGNED`. The `unassigned.reason` describes why
```

Comment:
If Elasticsearch can't move shards then they won't be `UNASSIGNED`, they'll stay assigned wherever they are. Also the `unassigned.reason` is rarely helpful in this situation, it really only describes why the shard became unassigned in the first place and doesn't usually say anything about how to fix it.